reflection.md

```diff
@@ -1,50 +1,72 @@
 # Reflection
+We managed to accomplish our original goals for this app in that we answered our research questions and delivered an app to
+handle the usage scenario that we set out to achieve. We now reflect upon our completed work. 
 
 ## Accomplishments
 
-- Our app runs very fast, which helps in selecting the movies you want to see. 
-- The UI of our is very simple and easily understood. 
-- Our app was able to accomplish what we proposed in milestone 3, with minor modifications based on the feedback we received from the course instructor.  
+- Our movie selection app is successfully built-in plotly/dashR and deployed using Heroku
+- The app is simple and easy to use 
+- The plots are interactive. Selecting a bar in the upper plot would highlight the same movie in the lower plot
+- The overplotting issue we had in the Python version of the app was successfully resolved by choosing a suitable plot
+- We were able to accomplish what we proposed in milestone 3, with some minor modifications based on the feedback we received  
 
 ## Limitations
 
-- Our app does not have the interactivity between the two plots that we proposed. This is partly due to the time constraints for the project, and based on our discussion with the course instructor.  
+- Our app does not have the interactivity between the two plots that we proposed. This is partly due to the time constraints for the project and based on our discussion with the course instructor.  
```

Comment:
We have this functionality now

Reply:
But maybe mention that it's not fully working as expected. For example, when new data is loaded, we can't reset the lower chart.

Reply:
Thanks for pointing it. I removed it. The limitations of the interactivity are already mentioned.

Reply:
nice. thanks.